modules/340-extended-monitoring/images/extended-monitoring-exporter/src/extended-monitoring.py

```diff
@@ -245,8 +245,8 @@ def _get_metrics():
     # iterate over namespaced objects in explicitly enabled via annotation Namespaces
     ns_list = (
         ns.metadata.name for ns in corev1.list_namespace().items
-        if ns.metadata.annotations
-        and EXTENDED_MONITORING_ENABLED_ANNOTATION in ns.metadata.annotations.keys()
+        if ns.metadata.labels
+        and EXTENDED_MONITORING_ENABLED_LABEL in ns.metadata.labels.keys()
```

Comment:
There is no need to get keys view

```suggestion
        and EXTENDED_MONITORING_ENABLED_LABEL in ns.metadata.labels
```


```python
>>> "a" in ({'a': 10}).keys()
True
>>> "a" in {'a': 10}
True
```

Reply:
Fixed!